README.md

```diff
@@ -15,9 +15,7 @@ Google's common Java, C++ and Javascript library for parsing, formatting, storin
   * ` PhoneNumberToCarrierMapper ` - provides carrier information related to a phone number.
 
 # Demo
-[Java Version](http://libphonenumber.appspot.com/) (running code r729 (on Google Code))
-
-[JavaScript Version](http://libphonenumber.googlecode.com/svn/trunk/javascript/i18n/phonenumbers/demo-compiled.html) (running JS code r729 (on Google Code))
```

Comment:
Thanks. I won't do this now as the build is borken anyway, so this will be submitted when it's all fixed.